### PR TITLE
fix: Remove coordinates from UI tap breadcrumbs

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

Removes x/y coordinates from automatic UI breadcrumbs. The `touch` breadcrumb's `view` field (and the `ui.lifecycle` breadcrumb's `window` field) used the default `UIView` description, which embeds the view's `frame = (x y; w h)`. We now build a safe view description (`<Class: pointer; opaque = ...>`) via `getUIViewDescription` that never includes the frame.

## :bulb: Motivation and Context

The tapped view's on-screen coordinates are a potential security risk: for apps with custom PIN code / passcode views, each key sits at a fixed position, so the coordinates can reveal which key a user tapped.

This is related to #8462. It does not solve that issue (Session Replay touch masking), but while investigating it we discovered that breadcrumbs also carry view coordinates, so we remove those here.

We remove the data entirely for now and may reintroduce it later behind an explicit opt-in if there is demand.

## :green_heart: How did you test it?

Added tests asserting the breadcrumb keeps the class name but no frame/coordinates, including a regression test that fails if the window field goes back to the raw `window.description`, plus edge-case coverage for `getUIViewDescription`. Ran `SentryBreadcrumbTrackerTests` (25 passing), plus `make analyze`.

## :pencil: Checklist

You have to check all boxes before merging:

- [x] I added tests to verify the changes.
- [x] No new PII added or SDK only sends newly added PII if `sendDefaultPII` is enabled.
- [x] I updated the docs if needed.
- [x] I updated the wizard if needed.
- [x] Review from the native team if needed.
- [x] No breaking change or entry added to the changelog.
- [x] No breaking change for hybrid SDKs or communicated to hybrid SDKs.
- [x] Public API changes reviewed by another Mobile SDK team member or implemented according to the [develop docs](https://develop.sentry.dev/) spec.
- [x] If I added a new public API, I also added it to the [SentryObjC wrapper](../develop-docs/SENTRY-OBJC.md).

### CODE REVIEW

File: Sources/Swift/Core/SwiftDescriptor.swift [hunk 1]
```diff
@@ -24,6 +24,24 @@ import UIKit
...
+    /// custom PIN/passcode views, so we never include the frame.
+    @objc
+    public static func getSanitizedViewDescription(_ view: UIView) -> String {
+        let className = getObjectClassName(view)
+        let pointer = Unmanaged.passUnretained(view).toOpaque()
+        var attributes: [String] = ["opaque = \(view.isOpaque)"]
+        if view.isHidden {
+            attributes.append("hidden = true")
+        }
+        return "<\(className): \(pointer); \(attributes.joined(separator: "; "))>"
```
Comment (itaybre) [inline Sources/Swift/Core/SwiftDescriptor.swift:41]: `m`: This may be a rabbit hole, but what if the class name contains an identifier to the value/number of the passcode? This may still be a PII leak.
Not sure how much we can protect here

Comment (philipphofmann) [reply Sources/Swift/Core/SwiftDescriptor.swift:41]: Other SDKs (sentry-java, react-native, sentry-dart) also send the view/class name in the breadcrumb, if I'm not mistaken. Unlike raw coordinates, view names and class names are a bit more obvious, so the SDK detects and sends them on purpose. I think it's okay to keep.

Comment (philipphofmann) [reply Sources/Swift/Core/SwiftDescriptor.swift:41]: @itaybre, if you're OK with this ⬆️ , can you maybe leave another review and approve if it's LGTM?

**Review by NinjaLikesCheez — APPROVED** [review]

File: Sources/Swift/Core/SwiftDescriptor.swift (hunk 1 above)
Comment (philipphofmann) [reply Sources/Swift/Core/SwiftDescriptor.swift:41]: I'm going to merge this PR as it already improves the situation. We can follow up if required.
